packages/snaps-sdk/package.json

```diff
@@ -8,9 +8,34 @@
   "sideEffects": false,
   "exports": {
     ".": {
+      "types": "./dist/types/index.d.ts",
       "import": "./dist/index.mjs",
-      "require": "./dist/index.js",
-      "types": "./dist/types/index.d.ts"
+      "require": "./dist/index.js"
+    },
+    "./jsx": {
+      "types": "./dist/types/jsx/index.d.ts",
+      "import": "./dist/jsx/index.mjs",
+      "require": "./dist/jsx/index.js"
+    },
+    "./jsx-runtime": {
+      "types": "./dist/types/jsx/jsx-runtime.d.ts",
+      "import": "./dist/jsx/jsx-runtime.mjs",
+      "require": "./dist/jsx/jsx-runtime.js"
+    },
+    "./jsx-dev-runtime": {
```

Comment:
What's the reason for exporting this under a different name too?

Reply:
Apparently build tools (like Vite for the WebdriverIO tests) expect a `jsx-dev-runtime` export.

Reply:
Interesting 😅